src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -1271,6 +1271,9 @@ int MPL_gpu_finalize(void)
 
     MPL_event_pool_destroy();
 
+    MPL_free(mask_contents.dev_id);
+    MPL_free(mask_contents.subdev_id);
```

Comment:
Were these leaks caught by the gpu-debug tests?

Reply:
Yeah we caught these when internally running the debug GPU tests for https://github.com/pmodels/mpich/pull/6451